### PR TITLE
Switch (temporarily) to using jpg for tiles on app side (AIC-383)

### PR DESCRIPTION
**Pull request by raigex**

### CODE REVIEW

File: map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt [hunk 1]
```diff
@@ -25,9 +28,18 @@ class GlideMapTileProvider(private val context: Context,
 
                 val path = "http://aic-mobile-tours.artic.edu/sites/default/files/floor-maps/tiles/floor$floor/zoom$zoom/tiles-$tileNumber.png"
                 return try {
+                    val bitmap = Glide.with(context).asBitmap().load(path).submit(TILE_SIZE.toInt(), TILE_SIZE.toInt()).get()
+                    val newBitmap = bitmap.copy(Bitmap.Config.RGB_565, true)
+                     val stream = ByteArrayOutputStream()
```
Comment (Cliabhach) [inline map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt]: Drop indent a little.

File: map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt [hunk 2]
```diff
@@ -25,9 +28,18 @@ class GlideMapTileProvider(private val context: Context,
 
                 val path = "http://aic-mobile-tours.artic.edu/sites/default/files/floor-maps/tiles/floor$floor/zoom$zoom/tiles-$tileNumber.png"
                 return try {
+                    val bitmap = Glide.with(context).asBitmap().load(path).submit(TILE_SIZE.toInt(), TILE_SIZE.toInt()).get()
```
Comment (Cliabhach) [inline map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt]: Add comment above this line so it's super clear how much of this code is temporary. Say something like `PNG decoding triggers crashes on some Android 8.1+ devices`.

File: map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt [hunk 3]
```diff
@@ -25,9 +28,18 @@ class GlideMapTileProvider(private val context: Context,
...
+                    val newBitmap = bitmap.copy(Bitmap.Config.RGB_565, true)
+                     val stream = ByteArrayOutputStream()
+                    newBitmap.compress(Bitmap.CompressFormat.JPEG, 100, stream)
```
Comment (Cliabhach) [inline map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt]: 100% so we don't lose any detail, gotcha.

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -10,7 +10,9 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme" >
+        android:theme="@style/AppTheme"
+        android:hardwareAccelerated="true"
```
Comment (Cliabhach) [inline app/src/main/AndroidManifest.xml]: Is this very important, or only kinda important?

Comment (raigex) [reply app/src/main/AndroidManifest.xml]: Rendering is faster in some cases

File: map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt (hunk 2 above)
Comment (raigex) [reply map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt]: done

File: map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt (hunk 1 above)
Comment (raigex) [reply map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt]: done

**Comment by Cliabhach** [comment]
It's looking much smoother.

**Pull request merged**